injective/assetlist.json

```diff
@@ -219,6 +219,48 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/WOSMO.png"
       }
+    },
+    {
+      "description": "GLTO-ERC20 on injective",
+      "denom_units": [
+        {
+          "denom": "ibc/072E5B3D6F278B3E6A9C51D7EAD1A737148609512C5EBE8CBCB5663264A0DDB7",
+          "exponent": 0
+        },
+        {
+          "denom": "glto",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/072E5B3D6F278B3E6A9C51D7EAD1A737148609512C5EBE8CBCB5663264A0DDB7",
```

Comment:
Sorry, I missed this before. I don't think an asset bridged from ethereum would be type: ics20, nor have 'ibc/...' as its base denom. Please correct the 0-exp denom, base, and type_asset.

Reply:
Not sure I understand, can you share a snippet of what it should look like

Reply:
@JeremyParish69 pushed up a change, hopefully it's correct